src/common/utils/time/time.mocha.ts

```diff
@@ -98,15 +98,15 @@ describe("Time", () => {
     };
 
     it("should hide year when just year is the same in all ticks", () => {
-      const scale = createScale(new Date("2019-01-01"), new Date("2019-02-01T10:00"));
+      const scale = createScale(new Date(Date.UTC(2019, 0, 2, 12, 34, 0)), new Date(Date.UTC(2019, 1, 1, 10, 0, 0)));
```

Comment:
I would say that new version is much less readable. What do you think about factory method in common/utils/time.ts for creating UTC dates based on formatted string?

Reply:
For sure I wouldn't add that to time.ts, it's test internals and shouldn't spill to app code. 

If we want to go for readability, I would use moment there. So function like 
```
(date: string) => moment.tz(date, "UTC").toDate()
```

Reply:
I agree with @adrianmroz, my intention was not to change the logic but to fix the unit test. I can do either helper function or a month enum (i.e JAN = 0, FEB = 1 and so on) to make it more readable.

Reply:
Using momentjs API looks really good! I don't think so that we need constants for month names, the code below is explicit and still readable, even without additional function/closure.

```
moment.tz("2019-01-01", "UTC").toDate
```